Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "servo-freetype-sys"
-version = "2.4.11"
```

Comment:
r- to that, it should be 2.4.12, this change won't even let you follow the freetype upstream version given 1101 > 12 anyway.

And actually, this changes introduce new external dependencies (cmake), so maybe it should even be bumped to 3.0.0, which would be great because that's an opportunity to stop following upstream versions, which is a bad idea anyway.